.github/CODEOWNERS

```diff
@@ -13,3 +13,4 @@
 /charts/pentaho-carte/ @ejcsid
 /charts/wjh-rechner/ @maximilian-zollbrecht
 /charts/zammad-ldap-sync/ @klml
+/charts/refarch-gateway/ @DanielOber @devtobi @simonhir
```

Comment:
I suggest referencing the @it-at-m/refarch-maintainer group here as well instead of hardcoded names :)

Reply:
Good idea for the codeowners but is not working in the Chart.yaml where also maintainers need to be defined.
I didn't come up with a great idea instead of listing people manually as links to groups are only working internally.
Do you have any idea on that?

Reply:
Ah, I See the problem. Helm only allows setting individual persons as maintainers inside the chart.yaml. That's bad. Nevertheless I would try to keep CODEOWNERS and helm chart maintainers in sync, which means you should add your own profile to chart maintainers as well. :)